src/directives/colony/outpostSK.ts

```diff
@@ -12,21 +15,74 @@ export class DirectiveSKOutpost extends Directive {
 	static directiveName = 'outpostSK';
 	static color = COLOR_PURPLE;
 	static secondaryColor = COLOR_YELLOW;
+	static containerRepairTheshold = 0.5;
+
+	memory: DirectiveSKOutpostMemory;
 
 	constructor(flag: Flag) {
 		super(flag, colony => colony.level >= 7);
 	}
+	
+	isCenterRoom(): boolean {
```

Comment:
this is unnecessary; use something like `const isCenterRoom = Cartographer.roomType(this.pos.roomName) == ROOMTYPE_CORE`

Reply:
done, check latest PR